src/box/CMakeLists.txt

```diff
@@ -60,6 +60,7 @@ lua_source(lua_sources lua/config/utils/tabulate.lua      config_utils_tabulate_
 
 if (ENABLE_CONFIG_EXTRAS)
     lua_source(lua_sources ${CONFIG_EXTRAS_DIR}/source/etcd.lua config_source_etcd_lua)
+    lua_source(lua_sources ${CONFIG_EXTRAS_DIR}/source/tarantool.lua config_source_tarantool_lua)
```

Comment:
There is no word about the new internal EE module in the commit message/pull request description. I think that it can be extracted into its own commit (within the PR). The PR title/description should reflect this change somehow.

Reply:
Updated.